src/display/api_compatibility.js

```diff
@@ -13,10 +13,10 @@
  * limitations under the License.
  */
 
-let compatibilityParams = Object.create(null);
-if (typeof PDFJSDev === "undefined" || PDFJSDev.test("GENERIC")) {
-  const { isNodeJS } = require("../shared/is_node");
```

Comment:
The commit message says "[...] `require` statements in code-paths not protected by pre-processor and/or run-time checks", but this one is inside a pre-processor check. Is this safe to change?

Reply:
This particular check passes in developer mode, note the `typeof PDFJSDev === "undefined" || ` part, and generally I do think that it's a good idea to run as much of the code as possible during development to help catch issues earlier (which is why I didn't change the condition here).

So yes, this change should be OK and the viewer still works locally with it :-)